docs/guide/apm-release-notes.asciidoc

```diff
@@ -2,6 +2,12 @@
 [[apm-release-notes]]
 == Release notes
 
+[float]
+==== APM version 7.0.0
+
+*APM Server*
+* Removal of deprecated Intake v1 API endpoints.
```

Comment:
There needs to be a blank line after `*APM Server*` for the bullet point to render correctly